serving/samples/helloworld-scala/.gitignore

```diff
@@ -0,0 +1,67 @@
+*#
```

Comment:
This is a pretty big `.gitignore`. Is it needed?

Reply:
I think I can prune it a bit more, it's a bit of a "catch all" .gitignore

Reply:
Can we remove it altogether, or are there intermediate outputs from `sbt` that we want to ignore?

Reply:
Absolutely, I've started from scratch and only added the entries strictly required.